setup.py

```diff
@@ -1,15 +1,17 @@
 # -*- coding: utf-8 -*-
 """
-Installs five executables:
+Installs the following command-line executables:
 
+    - ocrd_tesserocr_binarize
```

Comment:
ocrd-tesserocr-binarize etc. (dashes not underscores)

Reply:
Well spotted!